todoman/confspec.ini

```diff
@@ -14,6 +14,13 @@ color = option('always', 'auto', 'never', default='auto')
 # this option is not specified the system locale's is used.
 date_format = string(default='%x')
 
+# The date format used both for displaying times, and parsing input times.
+# Default is `%H:%M`.
+time_format = string(default='%H:%M')
+
+# The string used to separate date and time when displaying and parsing.
+dt_separator = string(default=' ')
```

Comment:
If one of `time_format` or `date_format` is an empty string, `dt_separator` is not used. Hence the `filter(bool, ...)` construction.